ibm_catalog.json

```diff
@@ -0,0 +1,125 @@
+        },
+        {
+          "title": "KMS encryption",
+          "description": "Will encrypt your data using supplied encryption keys. [Learn more](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-encryption)."
+        }
+      ],
+      "flavors": [
+        {
+          "label": "Compliant Instance and Bucket",
+          "name": "compliant-bucket-demo",
+          "index": 1,
+          "install_type": "fullstack",
+          "working_directory": "solutions/secure-bucket",
+          "compliance": {
+            "authority": "scc-wp-v1",
+            "profiles": [
+              {
+                "profile_name": "IBM Cloud Framework for Financial Services",
+                "profile_version": "1.7.0"
+              }
+            ]
+          },
+          "iam_permissions": [
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager"
+              ],
+              "service_name": "cloud-object-storage",
+              "notes": "Required to create COS bucket in existing instance."
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Reader"
+              ],
+              "service_name": "kms",
+              "notes": "Required to use existing KMS key for bucket encryption."
+            }
+          ],
+          "architecture": {
+            "features": [
+              {
+                "title": " ",
+                "description": "Configured to use IBM secure by default standards that can't be changed."
+              }
+            ],
+            "diagrams": [
+              {
+                "diagram": {
+                  "caption": "Secure COS Bucket Demo Architecture",
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cos/main/reference-architectures/regional-bucket.svg",
+                  "type": "image/svg+xml"
+                },
+                "description": "This architecture deploys a <b>Regional IBM Cloud Object Storage Bucket</b> in Brazil São Paulo with enforced security features including <b>KMS encryption</b>, object versioning, retention policies, and lifecycle management for secure, compliant cloud storage."
+              }
+            ]
+          },
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key"
+            },
+            {
+              "key": "bucket_name",
+              "required": true
+            },
+            {
+              "key": "existing_kms_key_crn",
+              "required": true
+            },
+            {
+              "key": "owner_contact_tag",
+              "required": true
+            },
+            {
+              "key": "technical_contact_tag",
+              "required": true
+            },
+            {
+              "key": "cos_instance_tags"
+            }
```

Comment:
you might want to use the custom array string picker for a better user experience on the tag inputs (assuming they are list of strings)